var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -114,6 +114,7 @@ class Mesa(MesonPackage):
         depends_on("libllvm@6:")
         depends_on("libllvm@:11", when="@:20")
         depends_on("libllvm@:12", when="@:21")
+        depends_on("libllvm@:15", when="@:22.1")
```

Comment:
Since `@:22.1` doesn't work with LLVM 15, should this instead be

```suggestion
        depends_on("libllvm@:14", when="@:22.1")
```

Reply:
The patch is restricted to libllvm-15 (ROCm 5.3.0) under the commit. [fc4b159](https://github.com/spack/spack/pull/33320/commits/fc4b159cd4c1e597004d996c974fb2c8934bd7bb)